advance_ai_agents/candidate_analyser/main.py

```
@@ -0,0 +1,197 @@
+from agno.models.openai.like import OpenAILike
+from agno.tools.github import GithubTools
+from agno.tools.exa import ExaTools
+from agno.tools.thinking import ThinkingTools
+from agno.tools.reasoning import ReasoningTools
+
+# Set wide layout
+st.set_page_config(layout="wide")
+
+# Load YAML prompts
+@st.cache_data
+def load_yaml(file_path):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)